scripts/zkai_attention_block_statement_bridge_gate.py

```
@@ -0,0 +1,717 @@
+    attention_outputs = _commitment(metrics.get("fixture_outputs_commitment"), "attention fixture outputs commitment")
+    attention_statement = _commitment(metrics.get("fixture_statement_commitment"), "attention statement commitment")
+    policy_commitment = _commitment(bridge_contract.get("policy_commitment"), "attention policy commitment")
+    attention_width = _int(metrics.get("value_width"), "attention value_width")
+    attention_steps = _int(metrics.get("steps"), "attention steps")
+
+    artifact = _dict(accumulator.get("accumulator_artifact"), "accumulator artifact")
+    preimage = _dict(artifact.get("preimage"), "accumulator preimage")
+    public_inputs = _dict(preimage.get("public_inputs"), "accumulator public inputs")
+    transcript = _list(preimage.get("verifier_transcript"), "accumulator verifier transcript")
+    if len(transcript) != 6:
+        raise AttentionBlockStatementBridgeError("d128 accumulator transcript must contain six slices")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)